README.md

```diff
@@ -129,7 +129,7 @@ research using the following BibTeX:
 
 ```
 @software{
-  pygmt_2024_10578540,
+  pygmt_2024_11062720,
   author       = {Tian, Dongdong and
```

Comment:
I think @yvonnefroehlich deserves a bump up the list now that she has made >200 commits to PyGMT :100: :100:, what do you think @seisman?

Output from `git shortlog HEAD...v0.1.0 -sne` at bc673bc9433b7d9c7e669d45620dd9d36b5558d5 (first 6 lines):

```
   718  Dongdong Tian <seisman.info@gmail.com>
   216  Wei Ji <23487320+weiji14@users.noreply.github.com>
   215  dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
   201  Yvonne Fröhlich <94163266+yvonnefroehlich@users.noreply.github.com>
   191  Will Schlitzer <schlitzer90@gmail.com>
   122  Michael Grund <23025878+michaelgrund@users.noreply.github.com>
```

Reply:
Sounds good. Done in 338e6b607.

Reply:
Also need to update the order in the Zenodo record.

Reply:
Yep, I'll remember to do that when uploading the release to Zenodo.